optaweb-vehicle-routing-backend/.dockerignore

```diff
@@ -2,4 +2,4 @@
 !target/*-runner
 !target/*-runner.jar
 !target/lib/*
-!target/quarkus-app/*
\ No newline at end of file
+!target/quarkus-app*/*
```

Comment:
Shell we add here local as well?

Reply:
No. We ignore everything (see `*` on line 1) except for the patterns starting with `!` so effectively only `target/quarkus-app-h2` and `target/quarkus-app-postgresql` will be copied and used to build the image.

See https://docs.docker.com/engine/reference/builder/#dockerignore-file for more details.